constants.yml

```diff
@@ -2,6 +2,7 @@
 javaDateFormat: "dd MMM yyyy"
 javaTimeFormat: "h:mm aa"
 javaDateTimeFormat: "dd MMM yyyy h:mm aa"
+javaScriptDateFormat: "DD MMM YYYY"
 javaScriptDateTimeFormat: "DD-MMM-YYYY HH:mm"
```

Comment:
Not quite sure why we have the hyphens in the date/time format, but figured I wouldn't change it